ansible/roles/piwik/tasks/setup-piwik.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Define release id
+  set_fact:
+    PIWIK_RELEASE_ID: "{{ [ansible_date_time.year, ansible_date_time.month, ansible_date_time.day, ansible_date_time.hour, ansible_date_time.minute, ansible_date_time.second] | join('') }}"
+
+- name: Download files
+  get_url: url={{ piwik_download_path }} dest={{ app_path }}/tmp/piwik.zip
```

Comment:
No real reason. I guess I didn't actually check for that URL and just followed the official setup instructions that use this as an example.